src/olympia/addons/tests/test_views.py

```diff
@@ -908,10 +916,9 @@ def _set_tested_url(self, param):
         self.url = reverse_ns('addon-version-list', kwargs={'addon_pk': param})
 
     def test_queries(self):
-        with self.assertNumQueries(13):
-            # 13 queries:
+        with self.assertNumQueries(11):
+            # 11 queries:
             # - 2 savepoints because of tests
-            # - 2 user and its groups
```

Comment:
This wasn't true: there was no query dealing with user & groups. Instead, the 2 extra queries that I've removed were in fact caused by license & license translation being fetched for each object. In other words, we had a massive scaling issue on that API.

Reply:
They were duplicated and miscounted before (you had one `licenses` and one `licenses translations` *for each version*. It just so happened so that were only 2 versions, and the comment misled us into thinking the 2 added queries were for users & groups).

They aren't anymore (you now have one `licenses` and one `licenses translations` *period*, regardless of how many versions are returned, thanks to the transformer), hence the lower number of queries.